prow/autobump.sh

```diff
@@ -73,21 +75,23 @@ extract-commit() {
   echo ${c##*-}
 }
 
-old_version=$(git show HEAD:prow/cluster/plank_deployment.yaml | extract-version)
-version=$(cat cluster/plank_deployment.yaml | extract-version)
-comparison=$(extract-commit "${old_version}")...$(extract-commit "${version}")
-echo -e "Pushing $(color-version ${version}) to ${user}:autobump..." >&2
+oncall=$(python2.7 -c 'import urllib2;import json;print json.loads(urllib2.urlopen("https://storage.googleapis.com/kubernetes-jenkins/oncall.json").read())["Oncall"]["testinfra"]')
 
-title="Bump prow from ${old_version} to ${version}"
+old_prow_version=$(git show HEAD:prow/cluster/plank_deployment.yaml | extract-version)
+prow_version=$(cat cluster/plank_deployment.yaml | extract-version)
+prow_comparison=$(extract-commit "${old_prow_version}")...$(extract-commit "${prow_version}")
+echo -e "Pushing prow $(color-version ${prow_version}) to ${user}:autobump..." >&2
+
+title="Bump prow (${old_prow_version} to ${prow_version}) and test images as necessary"
 git add -A
-git commit -m "${title}"
+git commit -m "Bump all images"
 git push -f "git@github.com:${user}/test-infra.git" HEAD:autobump
 
 echo "Creating PR to merge ${user}:autobump into master..." >&2
 bazel run //robots/pr-creator -- \
     --github-token-path="${token}" \
     --org=kubernetes --repo=test-infra --branch=master \
-    --title="${title}" --match-title="Bump prow to" \
-    --body="Included changes: https://github.com/kubernetes/test-infra/compare/${comparison}" \
+    --title="${title}" --match-title="Bump prow to and test images" \
```

Comment:
--match-title should be a subset of --title

Reply:
It was at some point! (and, relatedly, the titles were more unique).

I must've lost a change somewhere.